example/basic/11-scala-combined/bar/test/src-3/BarVersionSpecificTests.scala

```diff
@@ -3,7 +3,7 @@ import utest._
 object BarVersionSpecificTests extends TestSuite {
   def tests = Tests {
     test("test") {
-      assert(BarVersionspecific.text().contains("3.x"))
+      assert(BarVersionSpecific.text().contains("3.x"))
```

Comment:
I don't know why this wasn't blowing up before, but it's blowing up now, so I fixed it